### PR TITLE
Add @import UIKit; or @import Cocoa; for generated main.m

### LINKED ISSUE #4612: UIKit not added when linting with new linter
Should the new linter that generates the `App.xcworkspace` include `UIKit` for the iOS sample app? Again we are at the bleeding edge release that fixed this https://github.com/CocoaPods/CocoaPods/issues/4523 validator issue but now we are hitting this.

Or should the podspec include `UIKit` in its frameworks?

It only works when I manually add `UIKit` in the generated sample app...

### PR DESCRIPTION
**Pull request by dnkoutso**
closes #4612 

/cc @segiddins

### CODE REVIEW

File: lib/cocoapods/validator.rb [hunk 1]
```diff
@@ -421,7 +421,13 @@ def write_app_import_source_file(pod_target)
                                ''
                              end
         end
-        source_file.open('w') { |f| f << "@import Foundation;\n#{import_statement}int main() {}\n" }
+        source_file.open('w') do |f|
+          f << "@import Foundation;\n"
+          if consumer.platform_name == :ios || consumer.platform_name == :tvos
```
Comment (dnkoutso) [inline lib/cocoapods/validator.rb]: I can probably do that as a one liner...

**Comment by segiddins** [comment]
👍🏻

**Comment by segiddins** [comment]
Actually, does OS X need cocoa? @neonichu?

**Comment by dnkoutso** [comment]
I thought UIKit is only for iOS but ¯\_(ツ)_/¯

**Comment by segiddins** [comment]
Correct. But I think when linting for OS X we might also need to add a Cocoa import, but I'm not sure what's linked by default and I'm not at my computer at the moment.

**Comment by dnkoutso** [comment]
Ah I see. Let me check.

**Comment by dnkoutso** [comment]
Yes a new OSX project has a `ViewController.h` with `#import <Cocoa/Cocoa.h>`

**Comment by neonichu** [comment]
I think the Cocoa import is needed, yes.

**Comment by dnkoutso** [comment]
OK will add.

**Comment by dnkoutso** [comment]
Updated.

**Comment by dnkoutso** [comment]
Le green!

**Comment by neonichu** [comment]
LGTM

**Pull request merged**